crates/ruff_linter/src/rules/flake8_comprehensions/snapshots/ruff_linter__rules__flake8_comprehensions__tests__C419_C419.py.snap

```diff
@@ -103,8 +102,7 @@ C419.py:28:5: C419 [*] Unnecessary list comprehension
    |
 26 |   # Special comment handling
 27 |   any(
-28 |       [  # lbracket comment
-   |  _____^
+28 | /     [  # lbracket comment
```

Comment:
I find the new style less distracting but it's slightly less clear where the range starts. But I'd leave it as is because I think it's fine.